apps/src/codebridge/Editor/Editor.tsx

```diff
@@ -38,7 +44,23 @@ export const Editor = ({langMapping, editableFileTypes}: EditorProps) => {
 
   const editorConfigExtensions = useMemo(() => {
     if (file?.language && langMapping[file.language]) {
-      return [langMapping[file.language]];
+      const extensions: Extension[] = [langMapping[file.language]];
+      if (file.language === 'js') {
+        // eslint configuration
+        const config = {
+          ...js.configs.recommended,
+          languageOptions: {
+            globals: {
+              ...globals.browser,
```

Comment:
`globals.browser` makes it so the linter knows about things like `console` and doesn't report them as unknown variables.